executor/executor.go

```diff
@@ -39,6 +39,9 @@ func New(log zerolog.Logger, options ...Option) (*Executor, error) {
 	}
 	cfg.RuntimeDir = runtime
 
+	// todo: fix for windows
+	cfg.DriversRootPath = cfg.RuntimeDir + "/extensions"
```

Comment:
`filepath.Join` will use the correct path separator for the OS:

```suggestion
	cfg.DriversRootPath = filepath.Join(cfg.RuntimeDir, "extensions")
```